server/src/game/player/mod.rs

```diff
@@ -73,7 +73,7 @@ struct PlayerNightVariables{
 
     silenced: bool,
 
-    chosen_targets: Vec<PlayerReference>,
+    selection: Vec<PlayerReference>,
```

Comment:
selections?

Reply:
selection and selections mean almost the exact same thing